src/com/cloudogu/ces/cesbuildlib/Dockerfile.groovy

```diff
@@ -0,0 +1,35 @@
+package com.cloudogu.ces.cesbuildlib
+
+class Dockerfile {
+    private script
+
+    Dockerfile(script) {
+        this.script = script
+    }
+
+    /**
+     * Lints the Dockerfile with hadolint using a configuration file
+     *
+     * To configure hadelint, add a ".hadolint.yaml" file to your working directory
+     * See https://github.com/hadolint/hadolint#configure
+     *
+     * @param dockerfile Path to the Dockerfile that should be linted
+     * @param configuration Path to the hadolint configuration file
+     */
+    void lintWithConfig(String dockerfile = "Dockerfile", String configuration = ".hadolint.yaml"){
+        script.docker.image('hadolint/hadolint:latest-debian').inside(){
```

Comment:
Maybe extract the version into a variable and make it configurable. If something breaks in the latest version, we can then easily swap to an earlier version.

Reply:
Done